helpers/environmentCheck.js

```diff
@@ -0,0 +1,19 @@
+import {} from 'dotenv/config';
+
+const checkEnvironment = (app) => {
```

Comment:
The name of this file and function don't really correspond with it's function. Perhaps something like `useWebpackMiddleware` would be a more accurate name

Reply:
Thanks for the suggestion! Renamed the file and function to be `useWebpackMiddleware` for clarity.